src/components/Footer/Footer.js

```diff
@@ -0,0 +1 @@
+Footer.js
```

Comment:
Codacy found an issue: ['Footer' is not defined.](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5874309)